src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs

```diff
@@ -0,0 +1,28 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.OperationalInsights.Models;
+
+namespace Microsoft.Azure.Commands.OperationalInsights.Operations
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "OperationalInsightsOperation"), OutputType(typeof(IList<PSOperation>))]
```

Comment:
```suggestion
    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "OperationalInsightsOperation"), OutputType(typeof(PSOperation))]
```
The cmdlet does not return a list, instead, it returns the operation for many times, so the output type should be operation

Reply:
@isra-fel  is there a way to list all cmdlets that have "breaking changes"?

Reply:
We have limited types of breaking change. Do a full-text search of [these attributes](https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/breaking-changes-definition.md#cmdlets) you should be able to list all of them